packages/core/lib/commands/db/index.js

```diff
@@ -0,0 +1,46 @@
+const OS = require("os");
+const serveCommand = require("./commands/serve");
+
+const usage =
```

Comment:
The style of these; does it really match with the rest of the Truffle help messages? Maybe we should indent "Available sub-commands" and "serve \tStart..." a bit more...or we could list the subcommands like we list the options? So they are indented far in, what do you think?